agent-client/Cargo.toml

```diff
@@ -13,5 +13,4 @@ reqwest.workspace = true
 schemars.workspace = true
 serde.workspace = true
 serde_json.workspace = true
-workspace-hack = { version = "0.1", path = "../workspace-hack" }
-
+crucible-workspace-hack = { version = "0.1", path = "../workspace-hack" }
```

Comment:
In the omicron PR I saw these lines changed to:
`omicron-workspace-hack.workspace = true`

Can we do the workspace thing here as well, or will that not work for
other reasons?

Reply:
Oh I missed that, sorry!

Reply:
Fixed.